src/Components/PrevAndContinueButtons/PrevAndContinueButtons.tsx

```diff
@@ -0,0 +1,25 @@
+import { useNavigate, useParams } from 'react-router-dom';
+import PreviousButton from '../PreviousButton/PreviousButton';
+import { Button } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+type PrevAndContinueButtonsProps = {
+  currentStepId: number;
+};
+
+const PrevAndContinueButtons = ({ currentStepId }: PrevAndContinueButtonsProps) => {
+  const { uuid } = useParams();
+  const navigate = useNavigate();
+  const backNavigationFunction = () => navigate(`/${uuid}/step-${currentStepId - 1}`);
```

Comment:
Can we pass this function into this component so that if we need to do something different (like the household member question), we can have a different previous function?